### ISSUE TITLE
Lando scanners scans .lndo.site urls which are not in config

### ISSUE THREAD

**Issue by eldair** [lando/laravel#67]
When using lando start scanner starts scanning .lndo.site url for my site but it is not enabled or written anywhere in the config file. This started happening week or two ago.

![image](https://github.com/user-attachments/assets/386cc4ba-c2db-4d11-83bb-4fec72097dee)
![image](https://github.com/user-attachments/assets/27b1eb7c-eb61-4be9-90dc-b019016cca93)

**Comment by eldair** [comment]
In that case I prefer the bugged version 😄 Jokes aside, I don't use them and the real issue is the scanner taking a lot of time scanning those urls which are not setup so they're returning error for every test

**Comment by reynoldsalec** [comment]
@eldair in that case you could just disable the scanner: https://docs.lando.dev/core/v3/scanner.html#skipping

**Comment by eldair** [comment]
@reynoldsalec yeah, that is a good temp solution but won't that disable all scanners - even for my own urls?

**Comment by reynoldsalec** [comment]
Correct; I think the alternative would be to configure your app to use the `*.lndo.site` URLs. In my experience it's rare to have a local site _need_ to have a specific URL, but realize that exceptions exist.
